engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-errata-modal.html

```diff
@@ -105,6 +105,103 @@ <h4 data-block="modal-header">Content Host Errata Management</h4>
         </table>
       </div>
     </div>
+
+    <div name="errataDetails" ng-show="showErrata">
```

Comment:
I'm unclear why this is a better solution than simply linking to the erratum details page. If we really want to use modals, at least we could use a partial to reduce the risk of introducing bugs if something (i.e. the API) changes in the future. From a development point of view, it's just not DRY. Also, the erratum details page lists the repositories and content hosts affected as well, which I think (the repos) is useful information not currently provided by these modals.

I can see an argument for the modal in that it decreases context switching for our users. Maybe they just want to apply any applicable errata but want to just 'peek' at a few errata to see which packages are provided, etc.

Reply:
@akofink : So this modal logic with link redirects had these 2 pages [Errata Content host](https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/errata-content-hosts.html) and [Errata Details](https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/errata-details.html) which become dead code following this change. 

These 2 pages worked fine till 6.2 but once we moved to modals in 6.3, the redirects stopped working. 

Do you suggest turning these pages into modals or a simple redirect to the regular errata detail page etc..(We still have some context switching involved with links on the other modals. I don't know why errata was treated differently in the first place)

Reply:
Hmmm... I guess we should fix it to work as was intended in 3.4 (6.3) with the modals? It's kind of a UX question, so maybe @Rohoover has some input. We should definitely still support all the workflows we used to support in 3.0 (6.2) before we changed over to modals.